README.md

```diff
@@ -1,133 +1,126 @@
 Chitter Challenge
 =================
 
-* Challenge time: rest of the day and weekend, until Monday 9am
-* Feel free to use Google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday morning
+([Link to original README](./original-README.md))
 
-Challenge:
--------
+## Getting started
 
-As usual please start by forking this repo.
+_PostgreSQL should be installed_
 
-We are going to write a small Twitter clone that will allow the users to post messages to a public stream.
+```sh
+git clone git@github.com:dafuloth/chitter-challenge.git
+cd chitter-challenge
+bundle
 
-Features:
--------
+ruby app.rb
 
 ```
-STRAIGHT UP
 
-As a Maker
-So that I can let people know what I am doing  
-I want to post a message (peep) to chitter
+## Database setup
 
-As a maker
-So that I can see what others are saying  
-I want to see all peeps in reverse chronological order
+```sql
+psql
 
-As a Maker
-So that I can better appreciate the context of a peep
-I want to see the time at which it was made
+CREATE DATABASE chitter;
 
-As a Maker
-So that I can post messages on Chitter as me
-I want to sign up for Chitter
+\c chitter
+CREATE TABLE peeps (id SERIAL PRIMARY KEY,
+                    timestamp timestamptz NOT NULL DEFAULT now(),
+                    peep VARCHAR(140) NOT NULL);
 
-HARDER
+CREATE DATABASE chitter_test;
 
-As a Maker
-So that only I can post messages on Chitter as me
-I want to log in to Chitter
+\c chitter_test
+CREATE TABLE peeps (id SERIAL PRIMARY KEY,
+                    timestamp timestamptz NOT NULL DEFAULT now(),
+                    peep VARCHAR(140) NOT NULL);
 
-As a Maker
-So that I can avoid others posting messages on Chitter as me
-I want to log out of Chitter
+```
+Databases can be populated with the test data as follows:
 
-ADVANCED
+```sql
+psql chitter < ./db/migrations/02_peeps_test_data.sql
+psql chitter_test < ./db/migrations/02_peeps_test_data.sql
 
-As a Maker
-So that I can stay constantly tapped in to the shouty box of Chitter
-I want to receive an email if I am tagged in a Peep
 ```
 
-Technical Approach:
------
+## Challenges
+- I want the `timestamp` to be set automatically, based on when the peep is made. Determined that an easy way to achieve this is with default value `DEFAULT now()`.
+
+## Features
+
+My approach is to tackle the "_seeing peeps_" features first since they do not require signing up or posting (which requires signing up).
 
-This week you integrated a database into Bookmark Manager using the `PG` gem and `SQL` queries. You can continue to use this approach when building Chitter Challenge.
+### Design choices: The Single-Page Application
```

Comment:
As  a personal opinion, I prefer single page app too, for the reasons mentioned by you!